src/main/java/org/recordrobotics/charger/control/DoubleControl.java

```diff
@@ -24,6 +24,22 @@ public double getDriveLat() {
 		return -_gamepad2.getLeftX();
 	}
 
+	@Override
+	public ClawState getClawTurn() {
+		// Mimic button-like behavior
+		boolean cube = _gamepad2.getLeftTriggerAxis() > TRIGGER_THRESHOLD;
+		boolean cone = _gamepad2.getRightTriggerAxis() > TRIGGER_THRESHOLD;
+
+		// Cube takes precedence
+		if (cube) {
```

Comment:
I'm still not sure there's value in differentiating the input here between cube and cone..
How about start simple? "grab" or "neutral"
test it out, if it works, we're done. If it doesn't work, then we know this complexity is justified

Reply:
I believe this was already tested and claw group decided they need this.

Reply:
👍